packages/edit-site/src/components/header-edit-mode/index.js

```diff
@@ -75,8 +79,8 @@ export default function Header( { showIconLabels } ) {
 				'core/edit-site/toggle-list-view'
 			),
 			isVisualMode: getEditorMode() === 'visual',
-			settings: getSettings(),
 			blockEditorMode: __unstableGetEditorMode(),
+			homeUrl: getUnstableBase()?.home,
```

Comment:
This is much better. We were only using the URL setting, so there was no need to return the whole object.